css/styles.css

```diff
@@ -18,77 +16,44 @@ header {
   color: white;
 }
 
-input {
-  margin-left: 1px;
-  margin-top: 7px;
-  width: 100%;
-}
-
-.button {
-  margin-left: 300px;
-}
-
-@media screen and (width > 0) {
-  p {
-    margin-bottom: 5px;
-  }
-
-  @media screen and (width > 424px) {
-    p {
-      margin-bottom: 6px;
-    }
-  }
-
-  @media screen and (width > 767px) {
-    p {
-      margin-bottom: 5px;
-    }
-  }
+.full-column {
+  flex-basis: 100%;
 }
 
 @media screen and (width > 567px) {
   .column-half {
     flex-basis: 50%;
+    margin-bottom: 15px;
   }
 }
 
-@media screen and (width > 0) {
-  textarea {
-    width: 100%;
-    height: 100px;
-    margin-top: 5px;
-    margin-left: 1px;
-  }
-}
-
-@media screen and (width > 374px) {
-  textarea {
-    width: 100%;
-    height: 50px;
-    margin-top: 7px;
-  }
+input {
+  border-color: black;
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
 }
 
-@media screen and (width > 424px) {
-  textarea {
-    width: 100%;
-    height: 50px;
-  }
+textarea {
+  border-color: #4c2676;
+  border-radius: 10px;
+  border-width: 2px;
+  width: 100% !important;
+  height: 100%;
```

Comment:
You should be able to do this without using `!important`. Also this textarea's borders are much too rounded and the color of all of your inputs should be a light grey, and then this purple _on focus_